README.markdown

```diff
@@ -150,6 +150,15 @@ Supported options:
 
 (I.e. you want only interface declarations for your Protobuf types, then pass all three of `outputEncodeMethods`, `outputJsonMethods`, and `outputClientImpl` as `false`, i.e. `--ts_proto_opt=outputEncodeMethods=false,outputJsonMethods=false,outputClientImpl=false`.)
 
+The following options are very useful for NestJS and available if `nestJs` is set as `true`
```

Comment:
I have tried to give a little bit of insight on what the options do. I don't think we need to give specific examples here of what to do with the final interfaces, everyone coming from nestJS will know.

What I will do is add some context about what metadata is.

Reply:
Nice, thanks! I word-smithed this a bit, please check master and let me know if something doesn't sound right.